docs/pages/components/SearchField.mdx

```diff
@@ -72,6 +73,24 @@ We include trailing children to allow for additional input context to be provide
 </FieldWrap>
 ```
 
+### Controllable
```

Comment:
Nit: We've been calling this "controlled" in documentation.

```suggestion
### Controlled

`SearchField` can also be controlled, by setting the `value` prop and updating the value from event callbacks.
```

Reply:
Thanks, updated